packages/core/test/feedback.test.ts

```diff
@@ -322,12 +326,15 @@ describe('captureFeedback', () => {
     expect(typeof eventId).toBe('string');
     expect(span).toBeDefined();
 
-    const { spanId, traceId } = span!.spanContext();
+    const traceId = span!.spanContext().traceId;
 
-    expect(mockTransport).toHaveBeenCalledWith([
+    const [[envelope]] = mockTransport.mock.calls;
+    expect(mockTransport).toHaveBeenCalledWith(envelope);
+
+    expect(envelope).toEqual([
```

Comment:
The same as above, I think this change is not necessary.

Reply:
True, it wasn't before, but now it is, so I roll back the original implementation here 
  https://github.com/getsentry/sentry-react-native/pull/4568/commits/45ee6fadd66e5d745627d4eccdbab5698cbfb0c7